mysticeti-core/src/config.rs

```diff
@@ -49,6 +49,9 @@ pub struct Parameters {
     pub store_retain_rounds: u64,
     pub enable_cleanup: bool,
     pub synchronizer_parameters: SynchronizerParameters,
+    /// When detected connection latency is >= the `network_connection_max_latency`, then the connection breaks as best effort to fix any
+    /// transient connection issues.
+    pub network_connection_max_latency: Duration,
```

Comment:
Would it be better if the networking stack just handled this internally and did not expose this? It should maintain the health of the connection and reconnect if needed.

Reply:
I was really in between having this as a const in the network component or having as property, but ended up exporting as property to allow easier configuration - ultimately having this to some sort of config file. I'll keep this for now as a property and we can revert if we feel that is not needed.